test/extended/cli/debug.go

```diff
@@ -20,17 +20,29 @@ var _ = g.Describe("[cli] oc debug", func() {
 		err = oc.Run("create").Args("-f", templatePath).Execute()
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		exutil.WaitForAnImageStreamTag(oc, oc.Namespace(), "busybox", "latest")
+		exutil.WaitForAnImageStreamTag(oc, oc.Namespace(), "nodejs-ex", "latest")
 		o.Expect(err).NotTo(o.HaveOccurred())
 	})
 
-	g.It("should print the container entrypoint/command", func() {
+	g.It("should print the imagestream-based container entrypoint/command", func() {
```

Comment:
why did you have to introduce a build to do this test?  why wasn't the busybox imagestream definition sufficient?  also if this test is going to do a build it needs to be marked [Slow]